src/components/Urlsnarf/Urlsnarf.tsx

```diff
@@ -99,77 +145,107 @@ const Urlsnarf = () => {
-                        placeholder={"eg: eth0"}
+            {!loadingModal && (
+                <InstallationModal
+                    isOpen={opened}
+                    setOpened={setOpened}
+                    feature_description={description}
+                    dependencies={dependencies}
+                ></InstallationModal>
+            )}
+            <form
+                onSubmit={form.onSubmit((values) =>
+                    onSubmit({
+                        ...values,
+                        listenerInputType: selectedListenerInput,
+                    })
+                )}
+            >
+                {LoadingOverlayAndCancelButton(loading, pid)}
+                <Stack>
+                    <NativeSelect
+                        value={selectedListenerInput}
+                        onChange={(e) => setSelectedListenerInput(e.target.value)}
+                        label={"listener settings"}
+                        data={listeners}
                         required
+                        placeholder={"Interface or PCAP file"}
                     />
-                )}
-                {isListenerFile && (
+                    {isListenerInterface && (
+                        <TextInput
+                            {...form.getInputProps("listenerArgs")}
+                            label={"Interface"}
+                            placeholder={"eg: eth0"}
+                            required
+                        />
+                    )}
+                    {isListenerFile && (
+                        <TextInput
+                            {...form.getInputProps("listenerArgs")}
+                            label={"File path"}
+                            placeholder={"eg: /home/kali/Desktop/pcap.pcapng"}
+                            required
+                        />
+                    )}
                     <TextInput
-                        {...form.getInputProps("listenerArgs")}
-                        label={"File path"}
-                        placeholder={"eg: /home/kali/Desktop/pcap.pcapng"}
-                        required
+                        {...form.getInputProps("versusMode")}
+                        label={"Exclusion details"}
+                        placeholder={"eg: POST ; show every packet that excludes POST"}
                     />
-                )}
-                <TextInput
-                    {...form.getInputProps("versusMode")}
-                    label={"Exclusion Details"}
-                    placeholder={"eg: POST; Show every packets that excludes POST"}
-                />
-                <Button type={"submit"} color="cyan">
-                    Sniff
-                </Button>
-                {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
-                <ConsoleWrapper output={output} clearOutputCallback={clearOutput} />
-            </Stack>
-        </form>
+                    <Button type={"submit"} color="cyan">
```

Comment:
Remove color styling so it is consistent with the other tools. 
```suggestion
                    <Button type={"submit"}>
```